run.sh

```diff
@@ -65,8 +65,15 @@ case "$deployment_action" in
         deploy_ses
         ;;
     "deploy_caasp")
+        # this is CaaSP 3
         deploy_caasp
         ;;
+    "deploy_caasp4")
+        # this is CaaSP 4
+        source ${scripts_absolute_dir}/pre-flight-checks.sh check_caasp4_skuba_available
+        source ${scripts_absolute_dir}/pre-flight-checks.sh check_caasp4_terraform_available
```

Comment:
Our other pre-flight checks are called from the deployment actions scripts, can we move these there as well for consistency?

Reply:
Fixed